### PR TITLE
Create discussion improvements

### PR DESCRIPTION
**Pull request by gdaguin**
# Description

Create a discussion is now 100% over.

## Type of change

- Refactorization

## Material used:

Please delete options that are not relevant.
- iOS (Smartphone/Tablet) : iPhone 14 Pro Max / iPhone SE

## How Has This Been Tested?

We have two sheets : the first one to select the contacts for the discussion and the second one to choose the name of the discussion and see the information of the contacts previously added.

### Patrol

Please list here the tests created in Patrol for this pull request.

## Checklist:

- My code follows the style guidelines of this project
- I have performed a self-review of my own code
- I have commented my code, particularly in hard-to-understand areas
- I have made corresponding changes to the documentation
- My changes generate no new warnings
- I have added tests that prove my fix is effective or that my feature works
- New and existing unit tests pass locally with my changes
- Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: lib/ui/views/messenger/layouts/create_discussion_validation_sheet.dart [hunk 1]
```diff
@@ -0,0 +1,207 @@
...
+              Padding(
+                padding: const EdgeInsets.only(top: 15),
+                child: Align(
+                  alignment: Alignment.bottomCenter,
+                  child: Row(
+                    children: <Widget>[
+                      AppButtonTiny(
+                        AppButtonTinyType.primary,
+                        localizations.createDiscussion,
+                        const [0, 0, 0, 0],
```
Comment (redDwarf03) [inline lib/ui/views/messenger/layouts/create_discussion_validation_sheet.dart]: Could you create in `Dimens` class a constant about the margin with 0, 0, 0, 0 like `  static const List<double> none = <double>[0, 0, 0, 0];`

File: lib/ui/views/messenger/bloc/create_discussion_form.dart [hunk 2]
```diff
@@ -1,39 +1,38 @@
...
-  CreateDiscussionGroupFormState build() =>
-      const CreateDiscussionGroupFormState();
-
-  void setMemberAddFieldValue(AddPublicKeyTextFieldValue fieldValue) {
-    state = state.copyWith(memberAddFieldValue: fieldValue.toAccessRecipient);
-  }
+  CreateDiscussionFormState build() => const CreateDiscussionFormState();
 
-  void addMember(AccessRecipient member) {
+  void addMember(Contact member) {
```
Comment (redDwarf03) [inline lib/ui/views/messenger/bloc/create_discussion_form.dart:35]: Do you plan to remove completely `AccessRecipient` class ?

Comment (gdaguin) [reply lib/ui/views/messenger/bloc/create_discussion_form.dart:35]: I will check if we use it somewhere else, if not I will delete it.

Comment (gdaguin) [reply lib/ui/views/messenger/bloc/create_discussion_form.dart:35]: We are using it in NFT classes and in the add discussion sheet. So I am keeping it.

**Pull request merged**